osu.Game/Scoring/ScoreRank.cs

```diff
@@ -7,10 +7,10 @@ namespace osu.Game.Scoring
 {
     public enum ScoreRank
     {
-        [Description(@"F")]
+        [Description(@"D")]
```

Comment:
If `F` apparently doesn't exist, can this be deleted completely?

Reply:
yes, that's what i did in ppy/osu-resources#58

Reply:
Then why is it still here?

Reply:
oh, you meant the `ScoreRank.F`